third_party/github.com/hashicorp/golang-lru/simplelru/lru_interface.go

```diff
@@ -34,6 +34,6 @@ type LRUCache interface {
 	// Clears all cache entries.
 	Purge()
 
-  // Resizes cache, returning number evicted
-  Resize(int) int
+	// Resizes cache, returning number evicted
+	Resize(int) int
```

Comment:
👀

Reply:
I think this is because I ran one of the scripts in `./hack` directory...